charts/k8s-watcher/templates/deployment.yaml

```diff
@@ -41,9 +41,7 @@ spec:
         {{- include "metrics.init" . | indent 8 }}
       {{- end }}
       containers:
-        {{- if .Values.metrics.enabled }}
         {{- include "metrics.container" .        | indent 8 }}
```

Comment:
why did you remove the condition of if metrics is enabled?

Reply:
its already part of the metrics.container 
its redundant to do it twice